docs/get-started/quickstart.md

```diff
@@ -133,7 +115,7 @@ search_settings:
   default_search_fields: [title, body]
 
 indexing_settings:
-  commit_timeout_secs: 5
```

Comment:
Do you have a specific reason to modify this value?

Reply:
To be honest, I copied the config from https://github.com/quickwit-oss/quickwit/blob/main/config/tutorials/stackoverflow/index-config.yaml#L50
Because the document says stackoverflow-index-config.yaml is downloaded using the following command
curl -o stackoverflow-index-config.yaml https://raw.githubusercontent.com/quickwit-oss/quickwit/main/config/tutorials/stackoverflow/index-config.yaml
What value should we use? 5 or 10?

Reply:
10 is fine!

Reply:
Got it, thank you